src/lib/index.ts

```diff
@@ -2,6 +2,17 @@ import fetch from 'node-fetch';
 import path from 'path';
 import execSh from 'exec-sh';
 import { specmaticJarPathLocal } from '../config';
+import fs from 'fs';
+
+
+
+export const setSpecmaticEnvVariable = (name: string, value: string, fileName: string) => {
+  const file = require("../../" + fileName)
```

Comment:
can we try using import here

Reply:
I imported the path and using path.resolve for better results so that the function can perfectly find the file